src/Native/build.proj

```diff
@@ -72,7 +72,10 @@
     
     <Copy SourceFiles="$(PackagesDir)mlnetmkldeps\$(MlNetMklDepsPackageVersion)\runtimes\$(PackageRid)\native\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
           DestinationFolder="$(NativeAssetsBuiltPath)" />   
-    
+
+    <Copy Condition="'$(OS)' == 'Windows_NT'" SourceFiles="$(PackagesDir)mlnetmkldeps\$(MlNetMklDepsPackageVersion)\runtimes\$(PackageRid)\native\libiomp5md$(NativeLibExtension)"
```

Comment:
>Windows_NT [](start = 33, length = 10)

No support for Linux and MacOS?

Reply:
So this works on all platforms. For Linux and MacOS, we rely on openmp to be installed on the system. 

---
In reply to: [262795888](https://github.com/dotnet/machinelearning/pull/2867#discussion_r262795888) [](ancestors = 262795888)